### ISSUE TITLE
Refactor label code for various models

### ISSUE THREAD

**Issue by jpsember** [jpsember/java-ml#78]
The classification model will have different data types for the labels, depending upon whether they are ground truth labels (bytes) or logits (floats).
